Dockerfile

```diff
@@ -93,9 +93,9 @@ RUN mkdir $BUILD_OUTPUT_DIR && cd $BUILD_OUTPUT_DIR \
     ../ \
     && make -j$(nproc) all \
     # Copy CMake generated Testfile to be able to trigger ctest from bin directory
-    && cp tests/CTestTestfile.cmake bin/ \
+    && cp tests/CTestTestfile.cmake bin/ 
     # keep only required shared libraries and final binaries
-    && find . -maxdepth 1 ! -name "lib" ! -name "bin" -exec rm -rfv {} \;
+    # && find . -maxdepth 1 ! -name "lib" ! -name "bin" -exec rm -rfv {} \;
```

Comment:
I don't think commenting this helps.

Reply:
no, youŕe right it is a leftover from a test, fixed it.